language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/DefaultValueGenerationUtil.java

```diff
@@ -269,8 +274,7 @@ private static Optional<String> getDefaultValueForType(TypeSymbol bType, boolean
                 }
                 break;
             case TABLE:
-                TypeSymbol rowType = ((TableTypeSymbol) rawType).rowTypeParameter();
-                String rowValue = getDefaultValueForTypeDescKind(rowType).orElse("");
+                String rowValue = "";
```

Comment:
Is there a particular reason for this change? The row value is not always a string(` ""`). It should be based on the row type.

Reply:
In here with the previous approach the result will be `table [{}]`. After applying a code action the result is error nus because  we have an empty mapping constructor inside the square brackets. 

But if we provide the default value as `table []`. There will be no errors right?

Reply:
I see. +1